strawberry/subscriptions/__init__.py

```diff
@@ -1,2 +1,9 @@
+from typing_extensions import Literal
+
 GRAPHQL_TRANSPORT_WS_PROTOCOL = "graphql-transport-ws"
 GRAPHQL_WS_PROTOCOL = "graphql-ws"
+
+SubscriptionProtocolType = Literal["graphql-transport-ws", "graphql-ws"]
```

Comment:
I'd prefer the following, since it does not duplicate the constants. (should work, just tried it in the python interpretor) :)

```suggestion
SubscriptionProtocolType = Literal[GRAPHQL_TRANSPORT_WS_PROTOCOL, GRAPHQL_WS_PROTOCOL]
```

Reply:
yes! good idea!